src/net/sourceforge/kolmafia/textui/command/TrainFamiliarCommand.java

```diff
@@ -22,14 +23,14 @@ public void run(final String cmd, final String parameters) {
 
     String typeString = split[0].toLowerCase();
 
-    int type;
+    Goal type;
 
     if (typeString.equals("base")) {
```

Comment:
Might as well do an enhanced switch here that returns null as default and then error out on null. Only if you feel like it